src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -584,9 +583,8 @@ function showNewQuestionFrame(type) {
     $('#addNewQuestionTable').hide();
     $('#empty_message').hide();
     scrollToElement($('#questionTable-' + NEW_QUESTION)[0], { duration: 1000 });
-    $('#questionTable-' + NEW_QUESTION).find('.visibilityOptions').hide();
 
-    getVisibilityMessageIfPreviewIsActive($('#questionTable-' + NEW_QUESTION));
+    getVisibilityMessage($('#questionTable-' + NEW_QUESTION));
```

Comment:
Need to rename occurrences of `editTab` and `previewTab` throughout because the toggle button is now gone and these two names are no longer accurate